tests/models.py

```diff
@@ -865,6 +1311,14 @@ def test_process_file_with_none(self):
 
         self.assertEqual([], dagbag.process_file(None))
 
+    @timed(2)
```

Comment:
I am very wary of this -- given we run our tests on Travis this seems like a recipe for transient failures.

Reply:
i agree. i'll remove it. FWIW there're are a bunch of tests that also have brittle performance behavior:
see https://github.com/wongwill86/incubator-airflow/blob/master/tests/jobs.py#L2524 (test_scheduler_run_duration) and a few other tests in that file. the timeout in this test covers a bit more than simply bagging a dag too.

Are there any other ways people know of how to track performance statistics from build to build, like coveralls for performance? Perhaps not for this PR but at some other point.